### PR TITLE
Cleanup/fixes for initial official Kodi Add-on release.

### PR DESCRIPTION
**Pull request by ksooo**
Mostly minors/wordings, but also some small API function fixes.

### CODE REVIEW

File: pvr.sledovanitv.cz/addon.xml.in [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.sledovanitv.cz"
-  version="2.4.1"
-  name="PVR sledovanitv.cz(unofficial)"
+  version="1.0.0"
```
Comment (palinek) [inline pvr.sledovanitv.cz/addon.xml.in]: Should I remove/rename the existing tags from git? (does the kodi source/team even care about release tags in external addons?)

Comment (ksooo) [reply pvr.sledovanitv.cz/addon.xml.in]: For stable kodi branches we take care of release tags, yes. For addon versions for kodi master branch, no tags are needed

Comment (ksooo) [reply pvr.sledovanitv.cz/addon.xml.in]: Examples:

https://github.com/xbmc/repo-binary-addons/blob/master/pvr.octonet/pvr.octonet.txt

vs.

https://github.com/xbmc/repo-binary-addons/blob/Krypton/pvr.octonet/pvr.octonet.txt

Comment (palinek) [reply pvr.sledovanitv.cz/addon.xml.in]: Done. And created the 1.0.0 tag...

**Pull request merged**